### ISSUE TITLE
Systematic Weight Variations module

### ISSUE THREAD

**Issue by benkrikler** [benkrikler/fast-carpenter-github-test#13]
**Imported from [gitlab issue 13](https://gitlab.cern.ch/fast-hep/public/fast-carpenter/issues/13)**

The current implementation is too restrictive.  Should move to something that supports this sort of config:
```yaml
systematics:                                                                       
    weights:                                                                       
        pileUp: {nominal: puWeight, up: puWeightUp, down: puWeightDown}            
        totalMuonSF: {nominal: totalMuonSF} 
```

**Comment by benkrikler** [comment]
**On 2018-11-19 Benjamin Krikler (bkrikler) wrote:**

In addition, this module should only calculate the overall combination if and only if the event type is not data.

**Comment by benkrikler** [comment]
**On 2018-11-19 Benjamin Krikler (bkrikler) wrote:**

mentioned in commit 79047b80802d1e5351a4c8c150d22e4171541b4c

**Comment by benkrikler** [comment]
**On 2018-11-19 Benjamin Krikler (bkrikler) wrote:**

mentioned in merge request [gitlab:!16](https://gitlab.cern.ch/fast-hep/public/fast-carpenter/merge_requests/16)

**Comment by benkrikler** [comment]
**On 2018-11-19 Benjamin Krikler (bkrikler) wrote:**

closed via commit d28e76a2a6f78b7e282edc1bd353b7dba5166043
